gatling-maven-plugin/src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -223,6 +229,25 @@ public void execute() throws MojoExecutionException {
     }
   }
 
+  private void iterateBySimulations(Toolchain toolchain, List<String> jvmArgs, List<String> testClasspath, Collection<String> simulations) throws Exception {
+    Exception[] exc = new Exception[1];
+    for (String simulation : simulations) {
+      try {
+        executeGatling(jvmArgs, gatlingArgs(simulation), testClasspath, toolchain);
+      } catch (Exception e) {
```

Comment:
You didn't take my previous comment about not trapping `Exception` but just `GatlingSimulationAssertionsFailedException` into account. Other exceptions should be blocking and not trigger  `continueAfterFail` logic.